README.md

```diff
@@ -101,27 +101,29 @@ Table of Contents
 ### • Camera
 - [**Open Camera**](http://opencamera.sourceforge.net/) <sup>**[[F-Droid](https://f-droid.org/app/net.sourceforge.opencamera)]**</sup>
 - [**Simple Camera**](https://github.com/SimpleMobileTools/Simple-Camera) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.camera)]**</sup>
-
+- [**Photon Camera**](https://github.com/eszdman/PhotonCamera)
```

Comment:
I'm not quite sure whether this includes proprietary parts or not. Firebase etc. seem to be removed, but there's still an Google repository included.

Reply:
There's a F-Droid request already: https://github.com/eszdman/PhotonCamera/issues/46